opnode/contracts/abis/MultiDepositor.json

```diff
@@ -0,0 +1,13 @@
+{
+  "abi": [
+    {
+      "inputs": [],
+      "name": "deposit",
```

Comment:
What do you think about having this stuff in a test related file so that its someplace different than the contracts used in prod?